README.md

```diff
@@ -58,6 +90,14 @@ cairo-compile cairo_programs/abs_value_array.cairo --output cairo_programs/abs_v
 
 target/release/cairo-rs-run cairo_programs/abs_value_array_compiled.json --layout all
 ```
+### Using hints
+
+Currently, as this VM is under construction, it's missing some of the features of the original VM. Notably, this VM only implements a limited number of hints at the moment, while the [Python Cairo VM](https://github.com/starkware-libs/cairo-lang) allows for the implementation of user-defined hints.
```

Comment:
"allows for the implementation of user-defined hints." This line is a bit confusing. As the original VM allows users to run user-defined hints (any python code), not having to implement them. Implementing the hints is something we do from our side to support a similar behaviour to the original VM. Maybe we could say "allows running any hint/any python code". instead

Reply:
Solved!